exercises/scrabble-score/README.md

```diff
@@ -0,0 +1,45 @@
+Given a word, compute the scrabble score for that word.
```

Comment:
Add the following as the first line, then add another line after it:

```Markdown
# Scrabble Score

```

Reply:
That's been added.